config/checkstyle_checks.xml

```diff
@@ -424,6 +424,23 @@
       <message key="matchxpath.match"
                value="Avoid using deprecated method 'DetailAst.branchContains()'."/>
     </module>
+    <module name="MatchXpath">
+      <property name="id" value="noUsageOfGetFileContentsMethod"/>
+      <property name="query" value="//CLASS_DEF[
+          ./EXTENDS_CLAUSE[./IDENT[@text='AbstractCheck'
+                                   or @text='AbstractSuperCheck'
+                                   or @text='AbstractJavadocCheck'
+                                   or @text='AbstractClassCouplingCheck'
+                                   or @text='AbstractNameCheck'
+                                   or @text='AbstractParenPadCheck'
+                                   or @text='InputJavadocMetadataScraperAnnotationUseStyleCheck'
+                                   or @text='InputJavadocMetadataScraperRightCurlyCheck'
+                                   or @text='InputJavadocMetadataScraperWriteTagCheck'
+                                   or @text='InputJavadocMetadataScraperNoCodeInFileCheck']]
```

Comment:
please remove all inputs from validation.
Inputs are not real files that are input files for testing , they could be any by content and the more weird the better for us.